### PR TITLE
RHIDP-2510: Relocate existing content to new Installing and Upgrading RHDH titles

### PR DESCRIPTION
**Pull request by linfraze**
Based on the new RHDH content strategy, this PR relocates existing installation content from the Admin and Getting Started guides to 4 new titles that will live under the **Install and Upgrade** category on the docs splash page. 

**Versions:**
1.2+

**Jira link:**
https://issues.redhat.com/browse/RHIDP-2510

**Previews:** 
[NEW] [Installing RHDH on OCP](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-257/install-rhdh-ocp)
[NEW] [Installing RHDH on EKS](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-257/install-rhdh-eks)
[NEW] [Installing RHDH on AKS](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-257/install-rhdh-aks)
[NEW] [Installing RHDH in an air-gapped environment](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-257/install-rhdh-air-gapped)

NOTE: This is a **content restructure** effort. Additional improvements to the content itself are tracked by https://issues.redhat.com/browse/RHIDP-2701 and targeted for 1.3.

**Reviews:**
- [x] SME: @coreydaley @nickboldt 
- [x] QE: N/A
- [ ] Docs: @hmanwani-rh

### CODE REVIEW

File: assemblies/assembly-rhdh-integration-aks.adoc [hunk 1]
```diff
@@ -9,12 +10,6 @@ This integration requires the deployment of {product-short} on {aks-short} using
 * The Helm chart
 * The {product} Operator
 
-include::modules/admin/proc-rhdh-deploy-aks.adoc[leveloffset=+1]
```
Comment (hmanwani-rh) [inline assemblies/assembly-rhdh-integration-aks.adoc:12]: Is this PR also restructuring the integration content?

File: modules/admin/con-rbac-config-permission-policies-external-file.adoc [hunk 2]
```diff
@@ -36,7 +36,8 @@ You can add your `policy.csv` file to the {product-short} Helm Chart by creating
 * You are logged in to your {ocp-short} account using the {ocp-short} web console.
 * {product} is installed and deployed using Helm Chart.
 +
-For more information about installing the {product} on {ocp-short} using Helm Chart, see xref:proc-install-rhdh-ocp-helm_admin-rhdh[]. 
+//For more information about installing the {product} on {ocp-short} using Helm Chart, see xref:proc-install-rhdh-ocp-helm_{context}[].
```
Comment (hmanwani-rh) [inline modules/admin/con-rbac-config-permission-policies-external-file.adoc:39]: Will cross-references be handled in a separate PR?

File: modules/admin/proc-install-operator.adoc [hunk 3]
```diff
@@ -37,5 +37,6 @@ For enhanced security, deploy the {product} Operator in a dedicated default name
 
 .Additional resources
 
-* For more information about installing {product-short}, see xref:proc-install-rhdh-ocp-operator_admin-rhdh[Deploying {product} on {ocp-short} using the Operator].
+//* For more information about installing {product-short}, see xref:proc-install-rhdh-ocp-operator_{context}[Deploying {product} on {ocp-short} using the Operator].
```
Comment (hmanwani-rh) [inline modules/admin/proc-install-operator.adoc:40]: We can also update the cross-references in this PR using the ID or the assembly. What do you think?
@Gerry-Forde

File: modules/admin/proc-rbac-ui-create-role.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@ You can create a role in the {product} using the Web UI.
 .Prerequisites
 * You have an administrator role in the {product-short}.
 * You have installed the `@janus-idp/backstage-plugin-rbac` plugin in {product-short}. For more information, see link:{LinkPluginsGuide}[{NameOfPluginsGuide}].
-* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_admin-rhdh[].
+* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_{context}[].
```
Comment (hmanwani-rh) [inline modules/admin/proc-rbac-ui-create-role.adoc:9]: Will the RBAC content be placed under a different title?

File: modules/admin/proc-rbac-ui-delete-role.adoc [hunk 5]
```diff
@@ -11,7 +11,7 @@ The policies generated from a `policy.csv` or ConfigMap file cannot be edited or
 .Prerequisites
 * You have an administrator role in the {product-short}.
 * You have installed the `@janus-idp/backstage-plugin-rbac` plugin in {product-short}. For more information, see link:{LinkPluginsGuide}[{NameOfPluginsGuide}].
-* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_admin-rhdh[].
+* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_{context}[].
```
Comment (hmanwani-rh) [inline modules/admin/proc-rbac-ui-delete-role.adoc:14]: Please see my previous question.

File: modules/installation/proc-install-rhdh-ocp-operator.adoc [hunk 6]
```diff
@@ -19,7 +20,7 @@ For more information about creating a project in {ocp-short}, see link:https://d
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
 . Click *Create*.
-. Add <<Adding a custom application configuration file to {ocp-short} using the Operator, custom configurations>> for the {product} instance.
```
Comment (hmanwani-rh) [inline modules/installation/proc-install-rhdh-ocp-operator.adoc:22]: Are cross-references being replaced here? If so, please add the correct form of cross-references.

File: modules/installation/proc-rhdh-deploy-eks-using-operator.adoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [id='proc-rhdh-deploy-eks-using-operator_{context}']
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+= Installing {product} on {eks-name} ({eks-short}) using the Operator
```
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-eks-using-operator.adoc:3]: Why are we replacing the word 'deploy' with 'install'? Installing usually refers to the initial setup phase. Deployment, however, is a more comprehensive term that includes all the activities needed to make software or systems available and operational in their intended environment.

The procedure in this section covers all those activities and steps for the user. I would recommend retaining the term 'deploy' instead of substituting it with 'install.

File: titles/installation-rhdh/master.adoc [hunk 8]
```diff
@@ -4,3 +4,16 @@ include::artifacts/attributes.adoc[]
...
 :imagesdir: images
+
+// ocp deployment
+include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
+
+// aws eks deployment
+include::assemblies/assembly-install-rhdh-eks.adoc[leveloffset=+1]
+
+// aks deployment
+include::modules/installation/proc-rhdh-deploy-aks.adoc[leveloffset=+1]
```
Comment (hmanwani-rh) [inline titles/installation-rhdh/master.adoc:15]: Have we finalized our decision to use master.adoc instead of the individual title files? Or am I missing anything?

**Review by hmanwani-rh — COMMENTED** [review]
@linfraze Thanks for creating the PR. 
Please review the comments/questions below. I think we need to make a few decisions as team together before merging or sending this PR for SME/QE reviews.

Additionally, Is there a reason that we are removing all the deployment sections from the admin guide? Some of them seem specific to the integration part and would be better placed under the integration title.

**Comment by linfraze** [comment]
@hmanwani-rh yes, in [this Slack thread](https://redhat-internal.slack.com/archives/C06HW0TEQKS/p1717539357988179?thread_ts=1717529824.449699&cid=C06HW0TEQKS) I expressed that I am sharing this draft with the docs team for discussion prior to submitting for SME/QE review. I also explained in another thread why we are removing certain sections from the Admin guide  and why we are decoupling the Integration sections from the deployment sections. I also suggested in the thread that we coordinate on that separate since we are creating a new Integration guide as a separate project. So this PR is the first step in  these reviews, discussions, and collaborations.

File: assemblies/assembly-rhdh-integration-aks.adoc (hunk 1 above)
Comment (linfraze) [reply assemblies/assembly-rhdh-integration-aks.adoc:12]: This PR separates the integration content from the installation/deployment content > relocates the installation content to the new installation guide > leaves the integration content in the Admin guide. No structural or content changes are made directly to the sections that are specific to integration.

File: modules/admin/con-rbac-config-permission-policies-external-file.adoc (hunk 2 above)
Comment (linfraze) [reply modules/admin/con-rbac-config-permission-policies-external-file.adoc:39]: All impacted cross-links were fixed as part of this PR. Some have been temporarily removed and will be replaced when there is a URL available for the installation guide. NOTE: Impacted cross-links are xrefs that broke during the local build. It would be a good idea to perform a more extensive audit of relevant cross-links.

File: modules/admin/proc-rbac-ui-create-role.adoc (hunk 4 above)
Comment (linfraze) [reply modules/admin/proc-rbac-ui-create-role.adoc:9]: The RBAC content is not being relocated as part of this PR, but relocating the installation content did break an xref in the RBAC content. This fix enables a clean build. However, more extensive content and xref audits would be beneficial in the future.

File: modules/admin/proc-install-operator.adoc (hunk 3 above)
Comment (linfraze) [reply modules/admin/proc-install-operator.adoc:40]: Linking to the assembly, wherever that makes sense, is a great idea

File: modules/installation/proc-install-rhdh-ocp-operator.adoc (hunk 6 above)
Comment (linfraze) [reply modules/installation/proc-install-rhdh-ocp-operator.adoc:22]: This procedure, and this step in particular, needs to closer look, per RHIDP-2503

File: modules/installation/proc-rhdh-deploy-eks-using-operator.adoc (hunk 7 above)
Comment (linfraze) [reply modules/installation/proc-rhdh-deploy-eks-using-operator.adoc:3]: This is a great question, and one that has been asked in multiple forums. Several people have said that these terms are interchangeable and/or that "install" is acceptable for this use case. However, I continue to ask about and to welcome opinions on the "install" vs. "deploy" terminology topic.

File: titles/installation-rhdh/master.adoc (hunk 8 above)
Comment (linfraze) [reply titles/installation-rhdh/master.adoc:15]: We have not finalized any decisions about the use of master.adoc going forward. We did agree not to remove the workaround from existing titles, but applying the workaround to future titles is moot. It doesn't hurt either way, nor to have a mix of both approaches, but upcoming tooling changes might remove the use of the master.adoc file altogether, making both approaches futile.

File: assemblies/assembly-install-rhdh-ocp.adoc [hunk 9]
```diff
@@ -1,19 +1,31 @@
...
+** Updates automatically
+** Managed by Operator Lifecycle Management (OLM)
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+* *The {product} Helm chart*
+** Works seamlessly in both {ocp-short} and Kubernetes
+** Requires manual installation and management
```
Comment (gashcrumb) [inline assemblies/assembly-install-rhdh-ocp.adoc:16]: @linfraze would it make sense to expand this bullet to better contrast (if I'm reading this right) with the "Updates automatically" and "Managed by OLM" bullets mentioned in the operator installation method?  And maybe also with the first bullet, I wonder if it would be more clear to expand these two points out into separate bullets.

Comment (linfraze) [reply assemblies/assembly-install-rhdh-ocp.adoc:16]: @gashcrumb thanks for the feedback. It might be better to collapse the auto-update and OLM bullets into a single bullet rather than breaking the other one into two. Are you also suggesting that we break the bullet about working on both OCP and K8s into two? I'd like to better understand why that would give more clarity without adding unnecessary fluff. There might also be additional details that we can include that would further justify breaking the bullet into two.

Comment (gashcrumb) [reply assemblies/assembly-install-rhdh-ocp.adoc:16]: It seemed like these bullets sort of compare/contrast against the bullets listed under the operator based install method, and so I here think I'm equating clarity with an equal number of bullets for each method.  Which may not actually be the right direction, so figured I'd raise it and see what y'all thought :smile:

Comment (linfraze) [reply assemblies/assembly-install-rhdh-ocp.adoc:15]: Applied your feedback, thanks!

File: modules/installation/proc-rhdh-deploy-eks-using-operator.adoc [hunk 10]
```diff
@@ -1,6 +1,6 @@
...
 
 You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
```
Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-eks-using-operator.adoc:5]: based on input from Tim in RHIDP-2707, we don't want to encourage/support opertor on k8s without first installing OLM as a prereq.

Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-eks-using-operator.adoc:5]:
```suggestion
You can install the {product-short} Operator on {eks-short} using the https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to deploy your {product-short} instance in {eks-short}.
```

File: modules/installation/proc-rhdh-deploy-eks-using-operator.adoc (hunk 7 above)
Comment (nickboldt) [reply modules/installation/proc-rhdh-deploy-eks-using-operator.adoc:3]: I would agree that we're installing the operator or chart, then deploying 1 or more instances of RHDH. Chart and operator are just the installation mechanism; creating an instance is the deployment step.

File: modules/installation/proc-rhdh-deploy-eks-using-helm.adoc [hunk 11]
```diff
@@ -1,6 +1,6 @@
 [id='proc-rhdh-deploy-eks-using-helm_{context}']
 
-= Deploying {product} in {eks-name} ({eks-short}) using Helm Chart
+= Installing {product} in {eks-name} ({eks-short}) using Helm Chart
 
 When you deploy {product-short} in {eks-name} ({eks-short}) using Helm Chart, it orchestrates a robust development environment within the AWS ecosystem.
```
Comment (nickboldt) [inline modules/installation/proc-rhdh-deploy-eks-using-helm.adoc:5]:
```suggestion
When you install the {product-short} Helm Chart in {eks-name} ({eks-short}), it orchestrates the deployment of an {product-short} instance, which provides a robust developer platform within the AWS ecosystem.
```

File: assemblies/assembly-install-rhdh-ocp.adoc [hunk 12]
```diff
@@ -1,19 +1,30 @@
...
+= Installing {product} on {ocp-short}
+:context: install-rhdh-ocp
 
 You can install {product} on {ocp-short} using one of the following methods:
 
-* The Helm chart
-* The {product} Operator
+* *The {product} Operator*
+** Ready for immediate use in {ocp-short} after an administrator installs it with OperatorHub
+** Can be configured for use in Kubernetes
```
Comment (nickboldt) [inline assemblies/assembly-install-rhdh-ocp.adoc:10]:
```suggestion
** Requires preinstallation of Operator Lifecycle Management (OLM) to manage automated subscription updates, available out of the box in {ocp-short}
```

File: assemblies/assembly-install-rhdh-ocp.adoc [hunk 13]
```diff
@@ -1,19 +1,30 @@
...
+** Updates automatically with Operator Lifecycle Management (OLM)
```
Comment (nickboldt) [inline assemblies/assembly-install-rhdh-ocp.adoc:11]:
```suggestion
** Requires preinstallation of Operator Lifecycle Management (OLM) to manage automated subscription updates on Kubernetes

```

File: assemblies/assembly-install-rhdh-ocp.adoc [hunk 14]
```diff
@@ -1,19 +1,30 @@
...
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+* *The {product} Helm chart*
+** Ready for immediate use in both {ocp-short} and Kubernetes
+** Requires manual installation and management
+
+Use the installation method that best meets your needs and preferences.
```
Comment (nickboldt) [inline assemblies/assembly-install-rhdh-ocp.adoc:17]:
```suggestion
Use the installation method that best meets your needs, cluster type, and preferences.
```

File: titles/installation-rhdh/docinfo.xml [hunk 15]
```diff
@@ -4,7 +4,7 @@
 <subtitle>
 </subtitle>
 <abstract>
-    <para>{product} is an enterprise-grade platform for building developer portals. You can use multiple methods to deploy {product-short} on any supported platform.</para>
+    <para>{product} is an enterprise-grade platform for building developer portals. Administrative users can configure roles, permissions, and other settings to enable other authorized users to install {product-short} on any supported platform using either the Helm chart or the Operator.</para>
```
Comment (nickboldt) [inline titles/installation-rhdh/docinfo.xml:7]:
```suggestion
    <para>{product} is an enterprise-grade platform for building developer portals. Administrative users can configure roles, permissions, and other settings to enable other authorized users to deploy an {product-short} instance on any supported platform using either the Helm chart or the Operator.</para>
```

File: artifacts/attributes.adoc [hunk 16]
```diff
@@ -63,3 +63,6 @@
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/confuguring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
 
+//:LinkInstallGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/installation_guide_for_red_hat_developer_hub/index
```
Comment (nickboldt) [inline artifacts/attributes.adoc:66]:
```suggestion
:LinkInstallGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/installation_guide_for_red_hat_developer_hub/index
```

Comment (themr0c) [reply artifacts/attributes.adoc:66]: And even now, prefer a docs.redhat.com URL.

File: assemblies/assembly-install-rhdh-ocp.adoc (hunk 14 above)
Comment (themr0c) [reply assemblies/assembly-install-rhdh-ocp.adoc:17]: Specific to OCP, so we might omit cluster type. Or should we consider variants in the OpenShift family?

**Comment by linfraze** [comment]
Demo PR to show proposed structure for installation guide and to gather feedback. Will open a new PR when it is time to implement.

**Review by hmanwani-rh — COMMENTED** [review]
@linfraze Thank you for your efforts in this area. The titles look good. I still need to do a thorough review of the PR, but here are some initial comments on a few typos and minor issues I noticed at a high level.

File: assemblies/assembly-install-rhdh-ocp.adoc (hunk 14 above)
Comment (nickboldt) [reply assemblies/assembly-install-rhdh-ocp.adoc:17]: AFAIU the docs should just talk about "OpenShift" ... the various flavours of OCP vs. OSD vs. OS Local are all the same (basically).

**Comment by nickboldt** [comment]
I'm seeing more unrendered attributes in the code snippets. For example this:

```
. Push both images to the internal {ocp-short} image registry by running the following commands:
+
[source,terminal]
----
podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-chart-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-chart-version}
----
+
```

should have the `subs="attributes+"` declaration:

```
. Push both images to the internal {ocp-short} image registry by running the following commands:
+
[source,terminal,subs="attributes+"]
----
podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-chart-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-chart-version}
----
+
```

**Comment by linfraze** [comment]
@nickboldt @hmanwani-rh @Gerry-Forde this is existing content, so that means that these attributes are not rendering in the currently published version. Will fix here now and also create a Jira to check for any other instances in which the `subs="attributes+"` declaration needs to be applied to properly render this attribute in other code blocks throughout the docs.

**Comment by nickboldt** [comment]
> @nickboldt @hmanwani-rh @Gerry-Forde this is existing content, so that means that these attributes are not rendering in the currently published version. Will fix here now and also create a Jira to check for any other instances in which `[source,subs="attributes+"]` needs to be applied to properly render the attributes in other code blocks throughout the docs.

reported as https://issues.redhat.com/browse/RHIDP-3206 and this is something our new team members could tackle as it's a good first issue for getting used to contributing clean asciidoc.

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc [hunk 17]
```diff
@@ -113,58 +105,22 @@ upstream:
   volumePermissions:
     enabled: true
 ----
+--
 
-. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the `values.yaml` file created in the previous step:
+. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the values.yaml file created in the previous step:
```
Comment (linfraze) [inline modules/installation/proc-rhdh-deploy-eks-helm.adoc:109]: Not sure why this change was made - should the monospace formatting be reapplied?

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc [hunk 18]
```diff
@@ -113,58 +105,22 @@ upstream:
...
 +
-[source,terminal,subs="attributes+"]
+--
+[source,subs="attributes+"]
```
Comment (linfraze) [inline modules/installation/proc-rhdh-deploy-eks-helm.adoc]: Not sure why `terminal` was removed. Should it be added back?

File: modules/installation/proc-deploy-rhdh-instance-eks.adoc [hunk 19]
```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-install-rhdh-eks.adoc
+
+[id="proc-deploy-rhdh-instance-eks.adoc_{context}"]
+= Deploying the {product-short} instance on {eks-short} with the Operator
+
+.Prerequisites
+
+* A cluster administrator has installed the {product} Operator.
```
Comment (linfraze) [inline modules/installation/proc-deploy-rhdh-instance-eks.adoc:10]: [SME] can you approve the addition of this Prereq?

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc [hunk 20]
```diff
@@ -1,46 +1,36 @@
...
-//
-// * assemblies/assembly-rhdh-integration-aws.adoc
-//
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-[id="proc-rhdh-deploy-eks-using-helm_{context}"]
-= Deploying {product} on {eks-name} ({eks-short}) using Helm Chart
+[id='proc-rhdh-deploy-eks-helm_{context}']
+= Installing {product-short} on {eks-short} with the Helm chart
```
Comment (linfraze) [inline modules/installation/proc-rhdh-deploy-eks-helm.adoc:5]: [SME] can you confirm that this change is still technically accurate?

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 21]
```diff
@@ -1,12 +1,17 @@
-[id='proc-rhdh-deploy-eks-using-operator_{context}']
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+[id='proc-rhdh-deploy-eks-operator_{context}']
+= Installing {product-short} on {eks-short} with the Operator
 
-You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
+You can install the {product} Operator with or without the Operator Lifecycle Manager (OLM) framework.
```
Comment (linfraze) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc:7]: [SME] it would be nice to provide some more conceptual info to help the customer understand whether they should install with or without the OLM framework - can we talk about what add'l info we should include here?

File: modules/release-notes/con-relnotes-notable-features.adoc [hunk 22]
```diff
@@ -5,7 +5,9 @@ This section highlights new features in {product} {product-version}.
 
 == {product} Operator is now generally available (GA)
```
Comment (linfraze) [inline modules/release-notes/con-relnotes-notable-features.adoc:6]: Updating RN content is out of scope for this PR, but we need to create a Jira to remove this

File: modules/release-notes/con-relnotes-notable-features.adoc [hunk 23]
```diff
@@ -5,7 +5,9 @@ This section highlights new features in {product} {product-version}.
...
 
-You can use the {product} Operator to install {product-short} on your {ocp-short}cluster. For more information, see the link:{LinkAdminGuide}[Installing {product} using the Operator] section in the _Administration guide_.
+You can use the {product} Operator to install {product-short} on your {ocp-short} cluster.
+//For more information, see the link:{LinkAdminGuide}[Installing {product} using the Operator] section in the _Administration guide_.
+//replace with link to the installation guide.
```
Comment (linfraze) [inline modules/release-notes/con-relnotes-notable-features.adoc:10]: this info is no longer in the Admin guide; replace with a link to the proper title in a separate PR

File: assemblies/assembly-install-rhdh-ocp-helm.adoc [hunk 24]
```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-install-rhdh-ocp-helm"]
+= Installing {product} on {ocp-short} with the Helm chart
+:context: assembly-install-rhdh-ocp-helm
+
+You can install {product} on {ocp-short} by using the Helm chart with one of the following methods:
+
+* The {ocp-short} console
+* The Helm CLI
```
[the remark this answers is not in the document]
Comment (coreydaley) [reply assemblies/assembly-install-rhdh-ocp-helm.adoc:9]: We mean the Helm CLI.

File: assemblies/assembly-install-rhdh-ocp.adoc (hunk 14 above)
Comment (coreydaley) [reply assemblies/assembly-install-rhdh-ocp.adoc:17]: I agree with @nickboldt, it shouldn't matter unless they are using an air-gapped cluster.

File: modules/installation/proc-deploy-rhdh-instance-eks.adoc (hunk 19 above)
Comment (coreydaley) [reply modules/installation/proc-deploy-rhdh-instance-eks.adoc:10]: I think that looks good

File: modules/installation/proc-install-rhdh-ocp-helm-cli.adoc [hunk 25]
```diff
@@ -1,5 +1,8 @@
-[id="proc-installing-rhdh-on-openshift-with-helm-cli_{context}"]
-= Installing {product} on {ocp-brand-name} with Helm CLI
+// Module included in the following assemblies:
+// assembly-install-rhdh-ocp-helm.adoc
+
+[id="proc-install-rhdh-ocp-helm-cli_{context}"]
+= Deploying {product-short} from the {ocp-short} CLI with the Helm chart
```
[the remark this answers is not in the document]
Comment (coreydaley) [reply modules/installation/proc-install-rhdh-ocp-helm-cli.adoc:5]: We mean Helm CLI

File: modules/installation/proc-install-rhdh-ocp-helm-gui.adoc [hunk 26]
```diff
@@ -1,8 +1,10 @@
-[id='proc-install-rhdh-ocp-helm_{context}']
+// Module included in the following assemblies:
+// assembly-install-rhdh-ocp-helm.adoc
 
-= Deploying {product} on {ocp-short} using Helm Chart
+[id='proc-install-rhdh-ocp-helm-gui_{context}']
+= Deploying {product-short} from the {ocp-short} web console with the Helm Chart
 
-You can use a Helm chart in {ocp-brand-name} to install {product-short},  which is a flexible installation method.
+You can use a Helm chart to install {product-short} on the {ocp-brand-name} web console.
```
[the remark this answers is not in the document]
Comment (coreydaley) [reply modules/installation/proc-install-rhdh-ocp-helm-gui.adoc:7]: Yes, you can install using the Helm Chart through the web console instead of the Helm CLI

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc (hunk 20 above)
Comment (coreydaley) [reply modules/installation/proc-rhdh-deploy-eks-helm.adoc:5]: Yes, this looks good

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc (hunk 21 above)
Comment (coreydaley) [reply modules/installation/proc-rhdh-deploy-eks-operator.adoc:7]: Maybe we should include a link to https://olm.operatorframework.io/docs/ ?

File: modules/installation/proc-install-rhdh-ocp-helm-cli.adoc (hunk 25 above)
[the remark this answers is not in the document]
Comment (linfraze) [reply modules/installation/proc-install-rhdh-ocp-helm-cli.adoc:5]: Got it. So we should change this to _Deploying {product-short} on {ocp-short} with the Helm CLI?_

File: assemblies/assembly-install-rhdh-aks.adoc [hunk 27]
```diff
@@ -0,0 +1,15 @@
+[id='assembly-install-rhdh-aks']
+
+= Installing {product} on {aks-name} ({aks-short})
+:context: assembly-install-rhdh-aks
+
+You can install {product} on {aks-brand-name} ({aks-short}) using one of the following methods:
+
+* The {product} Operator
+* The {product} Helm chart
```
[the remark this answers is not in the document]
Comment (linfraze) [reply assemblies/assembly-install-rhdh-aks.adoc:9]: Enhance this content as part of RHIDP-2701

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 28]
```diff
@@ -1,12 +1,20 @@
-[id='proc-rhdh-deploy-eks-using-operator_{context}']
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+[id='proc-rhdh-deploy-eks-operator_{context}']
+= Installing {product-short} on {eks-short} with the Operator
```
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc:5]: Using the full product name in the headings ensures consistency and is also beneficial for SEO.
```suggestion
= Installing {product} on {eks-short} with the Operator
```

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc [hunk 29]
```diff
@@ -1,12 +1,20 @@
...
 
-You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
+You can install the {product} Operator with or without the Operator Lifecycle Manager (OLM) framework.
+
+.Additonal resources
```
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-eks-operator.adoc:9]: I believe we can omit the additional resources tag here. Typically, additional resources are added at the end of a section, but here the Installation on EKS section is just starting.

File: modules/installation/proc-deploy-rhdh-instance-eks.adoc [hunk 30]
```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-install-rhdh-eks.adoc
+
+[id="proc-deploy-rhdh-instance-eks.adoc_{context}"]
+= Deploying the {product-short} instance on {eks-short} with the Operator
```
Comment (hmanwani-rh) [inline modules/installation/proc-deploy-rhdh-instance-eks.adoc:6]:
```suggestion
= Deploying the {product} instance on {eks-short} with the Operator
```

File: modules/installation/proc-rhdh-deploy-eks-helm.adoc (hunk 20 above)
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-eks-helm.adoc:5]:
```suggestion
= Installing {product} on {eks-short} with the Helm chart
```

File: modules/installation/proc-rhdh-deploy-aks-helm.adoc [hunk 31]
```diff
@@ -1,21 +1,20 @@
-// Module included in the following assemblies:
-//
-// * assemblies/assembly-rhdh-integration-aks.adoc
+// Module included in the following assemblies
+// assembly-install-rhdh-aks.adoc
 
 [id="proc-rhdh-deploy-aks-helm_{context}"]
-= Deploying {product} on {aks-name} ({aks-short}) using the Helm chart
+= Deploying {product-short} on {aks-short} with the Helm chart
```
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-aks-helm.adoc:5]:
```suggestion
= Deploying {product} on {aks-short} with the Helm chart
```

File: modules/installation/proc-rhdh-deploy-aks-operator.adoc [hunk 32]
```diff
@@ -1,107 +1,13 @@
-// Module included in the following assemblies:
-//
-// * assemblies/assembly-rhdh-integration-aks.adoc
+// Module included in the following assemblies
+// assembly-install-rhdh-aks.adoc
 
-[id="proc-rhdh-deploy-aks-using-operator_{context}"]
-= Deploying the {product} on {aks-name} ({aks-short}) using the Operator
+[id='proc-rhdh-deploy-aks-operator_{context}']
+= Deploying {product-short} on {aks-short} with the Operator
```
Comment (hmanwani-rh) [inline modules/installation/proc-rhdh-deploy-aks-operator.adoc:5]:
```suggestion
= Deploying {product} on {aks-short} with the Operator
```

File: modules/installation/proc-install-rhdh-ocp-helm-gui.adoc [hunk 33]
```diff
@@ -1,8 +1,10 @@
-[id='proc-install-rhdh-ocp-helm_{context}']
+// Module included in the following assemblies:
+// assembly-install-rhdh-ocp-helm.adoc
 
-= Deploying {product} on {ocp-short} using Helm Chart
+[id='proc-install-rhdh-ocp-helm-gui_{context}']
+= Deploying {product-short} from the {ocp-short} web console with the Helm Chart
```
[the remark this answers is not in the document]
Comment (linfraze) [reply modules/installation/proc-install-rhdh-ocp-helm-gui.adoc:5]: Another reviewer recently suggested that we remove the product name from the headings altogether, which I disagreed with for SEO and search reasons. However, I think it is fine to use the full product name at the title / assembly level and to use the short product name at the module level. One reason we currently have some modules using the full product name is because they do not currently belong to an assembly, although I believe they should. 

If there is guidance on using long product names in all headings without exception, please share. I also think we should reach a consensus and establish a standard for product names in headings in the future, since every team member currently has a different opinion on the topic.

File: modules/installation/proc-rhdh-deploy-eks-operator.adoc (hunk 28 above)
Comment (linfraze) [reply modules/installation/proc-rhdh-deploy-eks-operator.adoc:5]: I do see that there is some inconsistencies with the headings in this file because they are not modularized, so I will change the product name in the sub-headings. 

https://issues.redhat.com/browse/RHIDP-3290 modularizes this content in the future
